chromadb/api/__init__.py

```diff
@@ -385,3 +389,148 @@ def max_batch_size(self) -> int:
         """Return the maximum number of records that can be submitted in a single call
         to submit_embeddings."""
         pass
+
+
+class ClientAPI(BaseAPI, ABC):
+    tenant: str
+    database: str
+
+    @abstractmethod
+    def set_tenant(self, tenant: str, database: str = DEFAULT_DATABASE) -> None:
```

Comment:
set_tenant must take a database since users could otherwise set_tenant() but not set the database correctly. Leading to an incorrect client state.

Reply:
That makes sense to me, I can remove the default arg